PendingReleaseNotes.md

```diff
@@ -2,15 +2,20 @@
 
 ## Breaking Changes
 
-- The removal of `CSI_ENABLE_READ_AFFINITY` option and its replacement with per-cluster
-read affinity setting in cephCluster CR (CSIDriverOptions section) in [PR](https://github.com/rook/rook/pull/13665)
+- Kubernetes v1.25 is the minimum supported Kubernetes version
+- The Rook operator config `CSI_ENABLE_READ_AFFINITY` was removed. v1.13 clusters that have modified
+  this value to be `"true"` must set the option as desired in each CephCluster as documented
+  [here](https://rook.github.io/docs/rook/v1.14/CRDs/Cluster/ceph-cluster-crd/#csi-driver-options)
+  before upgrading to v1.14.
```

Comment:
@Madhu-1 @iPraveenParihar please verify that this updated release note is correct.

Reply:
yea, its correct. But as I mentioned [here](https://github.com/rook/rook/pull/13809#discussion_r1547279487). It was a bug fix.